extensions/grpc/deployment/pom.xml

```diff
@@ -29,6 +29,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-security-deployment</artifactId>
```

Comment:
This basically means that the security extension is always added for a gPRC app. Shouldn't it be rather optional?